### PR TITLE
Travis with cypress

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: Dockerfile-dynamodb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM python:rc-buster
+RUN apt-get update && apt-get install -y default-jre && \
```
Comment (suzannehamilton) [inline Dockerfile-dynamodb:2]: It probably doesn't matter for the prototype, but do you think we pin the JRE and Dynamo DB versions once we're in Beta?

File: build.sbt [hunk 2]
```diff
@@ -20,5 +20,12 @@ libraryDependencies += "com.mohiva" %% "play-silhouette-persistence" % "5.0.7"
...
 libraryDependencies += "software.amazon.awssdk" % "aws-sdk-java" % "2.7.11"
 
-libraryDependencies += "org.scalatestplus.play" %% "scalatestplus-play" % "4.0.2" % Test
+//libraryDependencies += "org.scalatestplus.play" %% "scalatestplus-play" % "4.0.2" % Test
 libraryDependencies +=  "com.github.jarlakxen" %% "drunk" % "2.5.0"
+libraryDependencies ++= Seq(
+  "com.mohiva" %% "play-silhouette-testkit" % "5.0.7" % "test"
+)
+libraryDependencies ++= Seq(
+  "org.scalatestplus.play" %% "scalatestplus-play" % "4.0.2" % "test"
```
Comment (suzannehamilton) [inline build.sbt:28]: Why does this now need to be wrapped in `Seq`?

Also, does `% Test` do the same thing as `% "test"`? I'm not familiar enough with sbt to know.

File: tsconfig.json [hunk 3]
```diff
@@ -1,14 +1,16 @@
 {
   "compilerOptions": {
     "outDir": "./dist/",
-    "module": "es6",
+    "module": "commonjs",
```
Comment (suzannehamilton) [inline tsconfig.json:4]: Just to check - do sign-in and upload still work? IIRC, I had to add `"module": "es6"` to fix the loading of the AWS SDK in TypeScript.

File: build.sbt (hunk 2 above)
Comment (MancunianSam) [reply build.sbt:28]: I've no idea, it was like this in the documentation and changed it out of desperation at one point. It works though so I've deleted the commented line.

File: Dockerfile-dynamodb (hunk 1 above)
Comment (MancunianSam) [reply Dockerfile-dynamodb:2]: Yeah, we definitely need to lock them down at some point. Like you say, it's fine for now.

File: tsconfig.json (hunk 3 above)
Comment (MancunianSam) [reply tsconfig.json:4]: Oddly, it's now only working with commonjs and not with es6 but it looks like the app is still working so I think this is ok.

**Pull request merged**